src/components/internals.jl

```diff
@@ -387,6 +387,13 @@ function parse_dot_mod(ps::ParseState, is_colon = false)
             pushtotrivia!(ret, EXPR(:DOT, 1, 1))
             ps.nt = RawToken(kindof(ps.nt), ps.nt.startpos, ps.nt.endpos, ps.nt.startbyte + 1, ps.nt.endbyte, ps.nt.token_error, false, ps.nt.suffix)
         else
+            @static if VERSION > v"1.6-"
```

Comment:
Maybe we should check against an internal `CSTParser.SYNTAX_VERSION` instead so we can parse newer syntax with old Julia versions?

Reply:
Good idea, lets open a separate issue on this.